src/notification/notification-module-spec.ts

```diff
@@ -23,6 +22,10 @@ import {
 } from './flowdock-notify';
 
 import {
+  SlackNotify,
+} from './slack-notify';
+
```

Comment:
The practice seems to be all over the place around this project. But I'll switch this...